src/main/webapp/app/exercises/programming/manage/grading/testcase-analysis/testcase-analysis.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+    <h2>{{ 'artemisApp.programmingExercise.configureGrading.testAnalysis.title' | artemisTranslate: { exerciseTitle: exerciseTitle } }}</h2>
+    <table class="table">
+        <thead>
+            <tr>
+                <th scope="col" jhiTranslate="artemisApp.programmingExercise.configureGrading.testAnalysis.occurrence"></th>
+                <th scope="col" jhiTranslate="artemisApp.programmingExercise.configureGrading.testAnalysis.testCaseFeedback"></th>
+                <th scope="col" jhiTranslate="artemisApp.programmingExercise.configureGrading.testAnalysis.task"></th>
+                <th scope="col" jhiTranslate="artemisApp.programmingExercise.configureGrading.testAnalysis.testcase"></th>
+                <th scope="col" jhiTranslate="artemisApp.programmingExercise.configureGrading.testAnalysis.errorCategory"></th>
+            </tr>
+        </thead>
+        <tbody class="table-group-divider">
+            @for (item of feedbacks; track item) {
+                <tr>
+                    <td class="text-center">{{ item.count }} ({{ getRelativeCount(item.count) | number: '1.0-0' }}%)</td>
+                    <td>{{ item.detailText }}</td>
+                    <td class="text-center">{{ item.task }}</td>
+                    <td>{{ item.testcase }}</td>
+                    <td>Student Error</td>
+                    <!-- This is a placeholder, will be covered in follow up PRs -->
+                </tr>
+            }
+        </tbody>
+    </table>
+    <div>{{ 'artemisApp.programmingExercise.configureGrading.testAnalysis.totalItems' | artemisTranslate: { count: feedbacks.length } }}</div>
```

Comment:
Try `translateValues`

```suggestion
    <div jhiTranslate="'artemisApp.programmingExercise.configureGrading.testAnalysis.totalItems" [translateValues]="{ count: feedbacks.length }"></div>
```